src/vector/jitsi/index.ts

```diff
@@ -146,7 +160,7 @@ function createJWTToken() {
         room: "*",
         context: {
             matrix: {
-                token: widgetApi.openIDCredentials.accessToken,
+                token: openIdToken.access_token,
```

Comment:
```suggestion
                token: openIdToken.access_token, // eslint-disable-line camelcase
```

Reply:
ah yes, you would think that, wouldn't you. Somehow it doesn't matter - the linter is happy in this area and isn't complaining about `room_id` directly below either. I also can't see a block disabling the rule in the area, so I presume it is magic.